package.json

```diff
@@ -25,12 +28,12 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-register": "^6.9.0",
     "chai": "^4.0.2",
-    "graphql": "^0.13.0",
+    "graphql": "^14.0.2",
     "mocha": "^3.4.1",
     "rollup": "^0.45.0",
     "test-all-versions": "^3.3.2"
   },
   "peerDependencies": {
-    "graphql": "^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0"
+    "graphql": "^14.0.0"
```

Comment:
Can we keep the existing peer deps and just OR in 14.x? I don't think we should force people to use `graphql` 14.x yet.

Reply:
this'd likely be a major version bump for `graphql-tag` itself, i think. the reason i'm bumping the major here is that `graphql@14` is removing support for node@4, which would be a major breaking change for `graphql-tag` itself which previously supported node@4. 

what do you think?